libs/dh/message-archive/data-access-api/src/lib/dh-message-archive-data-access-api.module.ts

```diff
@@ -0,0 +1,142 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Injectable } from '@angular/core';
+import { ComponentStore, tapResponse } from '@ngrx/component-store';
+import { filter, map, Observable, switchMap, tap } from 'rxjs';
+import {
+  MessageArchiveHttp,
+  SearchCriteria,
+  SearchResultItemDto,
+} from '@energinet-datahub/dh/shared/domain';
+import { HttpErrorResponse, HttpStatusCode } from '@angular/common/http';
+import { ErrorState, SearchingState } from './states';
+interface SearchResultState {
+  readonly searchResult?: Array<SearchResultItemDto>;
+  readonly searchingState: SearchingState | ErrorState;
+  readonly continuationToken: string | null | undefined;
+}
+
+const initialState: SearchResultState = {
+  searchResult: [],
+  searchingState: SearchingState.INIT,
+  continuationToken: null,
+};
+
+@Injectable()
+export class DhMessageArchiveDataAccessApiModule extends ComponentStore<SearchResultState> {
```

Comment:
Suggestion: This is not a module and should therefore not be suffixed with `module` (notice this should also be reflected in the filename).  
```suggestion
export class DhMessageArchiveDataAccessApiStore extends ComponentStore<SearchResultState> {
```